schema/deploy/roles/identifier-minter/grants.sql

```diff
@@ -0,0 +1,21 @@
+-- Deploy seattleflu/schema:roles/identifier-minter/grants to pg
+-- requires: warehouse/identifier
+-- requires: roles/identifier-minter/create
+
+begin;
+
+-- This change is designed to be sqitch rework-able to make it easier to update
+-- the grants for this role.
+
+grant connect on database :"DBNAME" to "identifier-minter";
+
+grant usage
+   on schema warehouse
+   to "identifier-minter";
+
+grant select
```

Comment:
The identifier-minter needs to be able to insert into warehouse.identifier

Reply:
Thanks, I thought so but I couldn't explicitly find that in the SQL code when I was making this yesterday. I'm going to test the role now.